src/elements/content-sidebar/activity-feed/app-activity/AppActivity.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @flow
+ * @file Info component used by Keywords component
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+import getProp from 'lodash/get';
+import noop from 'lodash/noop';
+
+import InlineDelete from '../inline-delete';
+import { ReadableTime } from '../../../../components/time';
+import Tooltip from '../../../../components/tooltip';
+import { Link } from '../../../../components/link';
+import messages from '../../../common/messages';
+
+import './AppActivity.scss';
+
+const ONE_HOUR_MS = 3600000; // 60 * 60 * 1000
```

Comment:
Could be useful to share the threshold with `Comment` in case we ever change the underlying value.